src/app/components/create-group-dialog/create-group-dialog.component.html

```diff
@@ -0,0 +1,18 @@
+
+<h1 mat-dialog-title>Créer un groupe</h1>
```

Comment:
if it is a component, then it is injected inside a page so it won't be a H1 but h2 or higher number